src/core/Rule.ml

```diff
@@ -79,6 +79,7 @@ type formula =
    * https://github.com/returntocorp/semgrep/issues/1218
    *)
   | Inside of tok * formula
+  | Anywhere of tok * formula
```

Comment:
maybe add in comment (* alt: we could also do via where: - also: <formula> and consider it as an extra condition *)

Reply:
Sure